docs/bestpractice.rst

```diff
@@ -4,4 +4,33 @@
 Best Practices for Collecting Physiological Data
 ================================================
 
-Coming soon.
```

Comment:
Could you add a quick text at the beginning that says something in the lines "we're working on it, but here's the content we're going to write down" or similar?

Reply:
Yup!